### DISCUSSION TITLE
Fixing the contributors link in every pages

### DISCUSSION THREAD

**Discussion by MAHIN0093** [mdn/mdn-community#325] [Content]
I am not sure how to approach it. Couple of individual issues pop up here and there. But the link needs fixing everywhere. For example if you go to this random page [ https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/  ] and click on the MDN contributors list it takes you here [https://developer.mozilla.org/enUS/docs/Web/CSS/CSS_Flexible_Box_Layout/Mastering_Wrapping_of_Flex_Items/contributors.txt]
which contains an unclickable link to the actual github commit page. I feel this very weird and outdated. Not sure how to approach fixing it.

**Comment by yin1999** [comment]
The `contributors.txt` file is generate by the platform ([mdn/yari](https://github.com/mdn/yari)), see: https://github.com/mdn/yari/blob/2b309f2d905135b8dbb7cd73bdf05b150eac95ad/build/cli.ts#L191-L197.

I think this could be an entry point for you :)

**Comment by Josh-Cena** [comment]
The link is "unclickable" because it's just a text file. Do we really want to make it a proper HTML? Sounds like unnecessary bloat.

**Reply by MAHIN0093** [reply]
Don't we? I mean, it just disrupts the flow opening a txt file. Is there a specific reason that it has been made like that?
